src/include/rocm_smi_wrap.h

```diff
@@ -30,6 +30,8 @@ THE SOFTWARE.
 #include "nccl.h"
 
 ncclResult_t rocm_smi_init();
+ncclResult_t rocm_smi_getNumDevice(uint32_t* num_devs);
```

Comment:
Please update copyright year on this file

Reply:
done